### ISSUE TITLE
abandon the usage of numpy in the class of Solution

### ISSUE THREAD

**Issue by jiweiqi** [DENG-MIT/reactorch#18]
Such as
```
Ea = np.float64(
                        [self.model_yaml['reactions'][i]['rate-constant']['Ea'].split(' ')[0]])
```

**Comment by ZhiyuShi** [comment]
The original Solution class used numpy when the data type of Ea was str, and there were three such cases in all. What I need to do is to convert the data of "str list" type to the data of "tensor" type. Using "torch.Tensor()" directly will lead to an error. I first use the "map" function to convert the "str list" type to the "map" type, and then use the "list" function so that the data is converted to the "float list" type. Afterwards using "torch.Tensor()" will not lead an error.

**Comment by ZhiyuShi** [comment]
The "Ea" in the original yaml file is not data of "str" type. Therefore, the conclusion above was discovered after I modified the gri30.yaml file. My modification method was to select a reaction in the yaml file, and add " kJ" after "Ea" of the reaction (although the actual unit may not be kJ). I have made such modifications to all the three types of reactions, and the results now show that there is no problem with my modification of Solution.py.
